javascript/javascript2/week1/hyfBay/main.js

```diff
@@ -0,0 +1,32 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+console.log(products);
+
+function renderProducts(products) {
+    // your code here
+    let productsUl = document.createElement("ul");
+
+    products.forEach(product => {
+        let productLi = document.createElement("li");
+
+        let header = document.createElement("h1");
+        let productPrice = document.createElement("p");
+        let productRating = document.createElement("p");
+
+        header.innerHTML = product.name;
+        productPrice.innerHTML = `Price : ${product.price}`;
+        productRating.innerHTML = `Rating : ${product.rating}`;
+
+        productLi.appendChild(header);
+        productLi.appendChild(productPrice);
+        productLi.appendChild(productRating);
+
+        productsUl.appendChild(productLi);
+    });
+
+    let Body = document.querySelector("body");
+    Body.appendChild(productsUl);
```

Comment:
A variable with a capital letter. 😮  I guess that's OK but be consistent when naming variables. All your other variables are lowercase, e.g. `header`.
```suggestion
    let body = document.querySelector("body");
    body.appendChild(productsUl);
```

Reply:
True. Thanks